docs/content/4.sdk/2.getting-started/1.index.md

```diff
@@ -2,6 +2,326 @@
+
+  return {
+    props: {
+      products,
+    },
+  };
+}
+```
+```tsx[App Router]
+import { getSdk } from "../sdk.config";
+
+const sdk = getSdk();
+
+const { products } = await sdk.commerce.searchProduct();
+```
+```tsx[Client Side Rendering]
+import { useEffect, useState } from "react";
+import { useSdk } from "../hooks";
+
+export function ClientComponentUsingSDK() {
+  const sdk = useSdk();
+  const [cart, setCart] = useState([]);
+
+  useEffect(() => {
+    const fetchCart = async () => {
+      const newCart = await sdk.commerce.getCart({
+        cartId: '<cart_id>',
+        fields: 'code,guid,user(FULL)'
+      });
+      setCart(newCart);
+    };
+
+    fetchCart();
+  }, []);
+}
+```
+::
+
+Code above is just an example of how you can use the SDK in your application. For more information about the avaialble methods, please refer to the respective [Integration's documentation](https://docs.vuestorefront.io/integrations).
+
+That's it! You can now use VueStorefront SDK Module in your Next.js app ✨
+
+#tab-2
+
+## Installation
+
+In order to use the SDK, you'll need to install and configure the Vue Storefront SDK Nuxt module.
+
+1. Add `@vue-storefront/nuxt` dependency to your project
+
+```bash [Nuxt]
+# Using yarn
+yarn add --dev @vue-storefront/nuxt
+
+# Using pnpm
+pnpm add -D @vue-storefront/nuxt
+
+# Using npm
+npm install --save-dev @vue-storefront/nuxt
+```
+
+2. Add `@vue-storefront/nuxt` to the `modules` section of `nuxt.config.ts`
+
+```ts[nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: ["@vue-storefront/nuxt"],
+});
+```
+
+3. Configure the module
+
+To configure the module, use `vsf` key in the Nuxt configuration object and provide necessary information such as the Middleware instance address:
+
+```ts[nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: ["@vue-storefront/nuxt"],
+  vsf: {
+    middleware: {
+      apiUrl: "http://localhost:4000"
+    }
+  },
+});
+```
+
+## Initializing the SDK
+
+To use SDK in our application, we need to initialize it first. To do so, follow these steps:
+
+Create SDK Config file - `sdk.config.ts` in root directory of your project. 
+
+::info
+It is not necessary to name the file `sdk.config.ts` specifically or to keep it in the root of your project, but it is recommended to keep it consistent with the rest of the Vue Storefront project.
+::
+
+You should import all other SDK configuration components. See the example below illustrating the SDK configuration with SAP Commerce Cloud and Contentful modules.
+
+```ts[sdk.config.ts]
+import { sapccModule } from '@vsf-enterprise/sapcc-sdk';
+import {
+  contentfulModule,
+  ContentfulModuleType,
+} from "@vsf-enterprise/contentful-sdk";
+
+export default defineSdkConfig(
+  ({ buildModule, middlewareUrl, getCookieHeader }) => ({
+    commerce: buildModule(sapccModule, {
+      apiUrl: middlewareUrl + "/commerce",
+      requestOptions: {
+        headers: getCookieHeader,
+      },
```

Comment:
I don't think that the `sapccModule` takes `requestOptions` as an argument - imo you have it just in the unified module